server/api/cvrs.py

```diff
@@ -83,6 +89,11 @@ def process():
         contest_votes_allowed = []
         for contest_header in contest_headers:
             match = re.match(r"^(.+) \(Vote For=(\d+)\)$", contest_header)
+            if not match:
+                raise UserError(
+                    f"Invalid contest name: {contest_header}."
+                    + " Contest names should have this format: Contest Name (Vote For=1)."
```

Comment:
can we also say something like "And must match standardized contest file exactly", maybe including a "Expected <blah> from standardized contests, got <blah> from the CVR"?

Reply:
Hmm, we're not currently guaranteed to have the standardized contests file uploaded when the CVRs are uploaded, so we can't check that here.

Reply:
I thought we were? What are we matching against if not?

Reply:
We match against the contests that the AA selects from the standardized contests file, but we don't do that until later when generating sample size options. It's kind of awkward and something I'd like to discuss, but whatever we decide I'd like to handle it in a separate PR.

Reply:
Oh I see, just a generic regex. I feel like we should compare with the standardized contests file, but I defer to @mcchilders